[2023-03-20 14:49:47,685] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.1.create_postgres_table_business manual__2023-03-20T14:48:20.614346+00:00 [queued]>
[2023-03-20 14:49:47,733] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.1.create_postgres_table_business manual__2023-03-20T14:48:20.614346+00:00 [queued]>
[2023-03-20 14:49:47,735] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 14:49:47,736] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2023-03-20 14:49:47,736] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 14:49:47,792] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_postgres_table_business> on 2023-03-20 14:48:20.614346+00:00
[2023-03-20 14:49:47,799] {standard_task_runner.py:52} INFO - Started process 9961 to run task
[2023-03-20 14:49:47,808] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'to_posgrest_v4.1', 'create_postgres_table_business', 'manual__2023-03-20T14:48:20.614346+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/to_postgresql.py', '--cfg-path', '/tmp/tmpwhd0nbxl', '--error-file', '/tmp/tmpwdzkenz8']
[2023-03-20 14:49:47,811] {standard_task_runner.py:77} INFO - Job 50: Subtask create_postgres_table_business
[2023-03-20 14:49:48,116] {logging_mixin.py:109} INFO - Running <TaskInstance: to_posgrest_v4.1.create_postgres_table_business manual__2023-03-20T14:48:20.614346+00:00 [running]> on host 0b4cdb459521
[2023-03-20 14:49:48,443] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sakabuana31
AIRFLOW_CTX_DAG_ID=to_posgrest_v4.1
AIRFLOW_CTX_TASK_ID=create_postgres_table_business
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T14:48:20.614346+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T14:48:20.614346+00:00
[2023-03-20 14:49:48,534] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: final, Login: ***, Password: ***, extra: {}
[2023-03-20 14:49:48,617] {dbapi.py:225} INFO - Running statement: 
            create table if not exists business (
                business_id text PRIMARY KEY,
                name text,
                address text,
                city text,
                state text,
                postal_code text,
                latitude float,
                longitude float,
                stars float,
                review_count int,
                is_open int,
                attributes text,
                categories text,
                hours text
            ) 
        , parameters: None
[2023-03-20 14:49:48,714] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=to_posgrest_v4.1, task_id=create_postgres_table_business, execution_date=20230320T144820, start_date=20230320T144947, end_date=20230320T144948
[2023-03-20 14:49:48,782] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-20 14:49:48,979] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
